src/axom/slam/DynamicConstantRelation.hpp

```diff
@@ -107,12 +107,12 @@ class DynamicConstantRelation : public /*Relation,*/ CardinalityPolicy
         EmptySetTraits<FromSetType>::isEmpty(fromSet) ? 0 : fromSet->size())
     , m_fromSet(fromSet)
     , m_toSet(toSet)
+    , m_currentFromSize(fromSet == nullptr ? 0 : m_fromSet->size())
   {
-    m_relationCardinality = CardinalityPolicy::size(0);
-    m_relationsVec.resize(m_relationCardinality * fromSet->size(), INVALID_INDEX);
+    updateSizes();
   };
 
-  ~DynamicConstantRelation() {};
+  ~DynamicConstantRelation() = default;
```

Comment:
It may be worthwhile to delete this destructor entirely; a user-defined destructor will inhibit generation of implicit move constructors.